leverage/modules/aws.py

```
@@ -210,6 +217,13 @@ def login(cli):
     if exit_code := cli.sso_login():
         raise Exit(exit_code)
 
+    # Refresh all account credentials if requested
+    if refresh_all:
+        try:
+            refresh_all_accounts_credentials(cli, force_refresh=False)
+        except ExitError as e:
+            raise Exit(e.code)
```

Pylint (4.0.1)

[error] Instance of 'ExitError' has no 'code' member

(E1101)